README.md

```diff
@@ -6,7 +6,7 @@ Configuration for providing pre-build [pysqlcipher3](https://github.com/rigglema
 
 This is a collections of patches and scripts to build wheels for rotki and publish them on PyPI.
 
-It builds wheels for CPython 3.9 for the following architectures:
+It builds wheels for CPython 3.9-3.10 for the following architectures:
```

Comment:
This is actually wrong it only builds for 3.10 now.

Reply:
All right interesting. So we will always be moving to next version and lose the ability to build wheeld for the old one? Just have them in pypi?

Reply:
Yes, so the idea is that we only build wheels for that version each time we move the application to a new Python version.

And all the previous versions are in pypi so if you want a wheel for the older version you can still use the older wheels.